hogben/models/samples.py

```diff
@@ -375,3 +408,8 @@ def similar_sld_sample_2():
 
         # Close the plots.
         plt.close('all')
+
+
+
+if __name__ == '__main__':
+    run_main()
```

Comment:
I'm not 100% sure, but I think there is  a missing empty line at the end of this file